app/src/components/main/notification/notificationContainer/notificationItem/notification.jsx

```diff
@@ -8,32 +8,38 @@ const cx = classNames.bind(styles);
 
 const messages = defineMessages({
   successLogin: { id: 'Notification.successLogin', defaultMessage: 'Signed in successfully' },
+  failureLogin: { id: 'Notification.failureLogin', defaultMessage: 'Wrong username or password' },
```

Comment:
As I see on our backbone version, in case of failure login attempt we show "An error occurred while connecting to server: " + error message returned in response from WS